File: app/src/main/java/com/codenicely/services/feelbazaar/vendorapp/course_detail/view/CourseDetailView.java
package com.codenicely.services.feelbazaar.vendorapp.course_detail.view;

import com.codenicely.services.feelbazaar.vendorapp.course_detail.data.CourseDetail;
import com.codenicely.services.feelbazaar.vendorapp.helper.Progressive;
import com.codenicely.services.feelbazaar.vendorapp.helper.Toaster;

public interface CourseDetailView extends Progressive, Toaster {
    void setData(CourseDetail data);
}

File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/course_detail/common/view/CourseDetailView.kt
package org.wikiedufoundation.wikiedudashboard.ui.course_detail.common.view

import org.wikiedufoundation.wikiedudashboard.ui.course_detail.common.data.CourseDetail
import org.wikiedufoundation.wikiedudashboard.util.Progressive
import org.wikiedufoundation.wikiedudashboard.util.Toaster

interface CourseDetailView : Progressive, Toaster {
    fun setData(data: CourseDetail)
}
